socketio/examples/readme.rs

```diff
@@ -10,6 +10,7 @@ fn main() {
         match payload {
             Payload::String(str) => println!("Received: {}", str),
             Payload::Binary(bin_data) => println!("Received bytes: {:#?}", bin_data),
+            Payload::Number(num) => println!("Received: {}", num),
```

Comment:
nit: what value does this add?

Reply:
currently can not emit simple number payload